src/azure-cli/azure/cli/command_modules/appconfig/snapshot.py

```diff
@@ -95,7 +95,7 @@ def show_snapshot(cmd,
 
 def list_snapshots(cmd,
                    snapshot_name=SearchFilterOptions.ANY_KEY,
-                   status=SearchFilterOptions.ANY_KEY,
```

Comment:
May I ask modifying this default value may cause a breaking change for customers?

Reply:
@zhoxing-ms  No please. The updated default value returns the same result.